packages/api/src/storage.ts

```diff
@@ -0,0 +1,55 @@
+import fs from 'fs'
+import path from 'path'
+
+import type * as Api from '../../shared/src/utils/api'
+import {createLogger} from '../../shared/src/utils/logging'
+
+export class LocalApiStore {
+  _localFsPath: string
+  _messagesPath: string
+  constructor(localPath: string, folder?: string) {
+    folder = folder || 'App_File_Storage'
+    this._localFsPath = path.join(localPath, folder)
+    // Init the directories
+    fs.mkdirSync(path.join(this._localFsPath, 'messages'), {recursive: true})
```

Comment:
@patrickhulce does using `fs` inside of electron make sense, or is there some `electron-fs` package I should use.

Reply:
oh no worries `fs` is totally fair game 👍 inside of our workers and main process you can think of it exactly like node